.github/workflows/conformance-aks.yaml

```diff
@@ -2,26 +2,24 @@ name: ConformanceAKS (ci-aks)
 
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
-  issue_comment:
-    types:
-      - created
+  workflow_dispatch:
+    inputs:
+      PR-number:
+        description: "Pull request number."
+        required: true
+      context-ref:
+        description: "Context in which the workflow runs. Will be either the PR branch itself (default), or the PR target branch (if user has write privileges on the repository)."
```

Comment:
`if user has write` - > `if user has no write`?

Reply:
Yeah, you are correct. And it is also confusing to read... What do you think about rephrasing it like this instead? I think this makes it easier to understand.
> Context in which the workflow runs. If PR is from a fork, will be the PR target branch. If PR is NOT from a fork, will be the PR branch itself.

Reply:
I like it.

Reply:
Slightly more detailed version:
> Context in which the workflow runs. If PR is from a fork, will be the PR target branch (general case). If PR is NOT from a fork, will be the PR branch itself (this allows committers to test changes to workflows directly from PRs).

Reply:
One more:
```
Workflow context:
 - PR branch itself for PR from the origin (for contributors with write privileges to test changes)
 - target branch for PR from a fork (general case)
```

Reply:
Can't do lists -- it's a single line description 😢